package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -376,7 +381,7 @@ dhcp_host_add() {
 		addrs="${ip:+,$ip}${hostid:+,[::$hostid]}"
 		xappend "--dhcp-host=$macs${duids:+,$duids}$hosttag$addrs$nametime"
```

Comment:
In case dnsmasq is compiled with IPv6 support the configured clientId is ignored ?
Imo the clientId should be added as well in this case

Reply:
Not in case it's compiled with but just for IPv6.
This works fine with the current stable x86.

Reply:
To avoid misunderstanding the check "$DNSMASQ_DHCP_VER -eq 6 " verifies if dnsmasq is compiled with DHCPv6 support on top of DHCPv4 support; therefore I fail to understand why the configured clientid should be ignored in case DHCPv6 support is enabled next to DHCPv4